e2e/config/browserstack.conf.js

```diff
@@ -9,7 +9,6 @@ let capabilities = {
   "device": "Google Pixel 3",
   "os_version": "9.0",
   "app": process.env.BROWSERSTACK_APP_ID,
-  "browserstack.local": true,
```

Comment:
what does this do?

Reply:
It allows you to connect to processes running on the machine which is running the tests (in this case circleci). We aren't doing that anymore and it's just one more thing which can go wrong on the test when the setup code fails so I removed it.